src/app/components/task/action/add/add-task-form.html

```diff
@@ -47,7 +47,7 @@
                 </div>
                 <div class="form-element">
                     <bl-form-field>
-                        <bl-duration-picker formControlName="retentionTime" label="Retention time" [required]="true" [allowUnlimited]="false" [defaultDuration]="7"></bl-duration-picker>
```

Comment:
Seems like you are removing the functionality instead of fixing it here no?

Reply:
True true.